labman/db/composition.py

```diff
@@ -1012,23 +1012,32 @@ class PoolComposition(Composition):
     _composition_type = 'pool'
 
     @staticmethod
-    def list_pools():
-        """Generates a list of pools with some information about them
+    def get_components_type(components):
+        """Return class name of components of pool
 
-        Returns
-        -------
-        list of dicts
-            The list of pool information with the structure:
-            [{'pool_id': int, 'external_id': string}]
+        Parameters
+        ----------
+        components: list-like of Compositions
         """
+
+        # This logic is taken from the original
+        # process.PoolingProcess.generate_pool_file method
+        # TODO: Someday: I don't like how this checks only the *first* of the
```

Comment:
Should the TODO be issues in the repo and add the issue number to this comment for easier tracking? This questions applies to all the TODOs in this PR.